.github/workflows/ci.yml

```diff
@@ -75,7 +75,7 @@ jobs:
 
   build_test_all_windows:
     needs: setup
-    if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'build_all')
+    if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'build_test_all_windows')
```

Comment:
FYI, you can opt-in to running jobs like this one by following
https://github.com/openxla/iree/blob/main/docs/developers/developing_iree/contributing.md#ci-behavior-manipulation
(Append `ci-extra: build_test_all_windows` to your PR description for this)

Reply:
Ah, thank you!